database/mongo/__tests__/unit.test.mjs

```diff
@@ -117,3 +117,73 @@ describe("Mongo Database Unit Functions. #mongo_unit #db", () => {
     expect(true).toBeTruthy()
   })
 })
+
```

Comment:
Not sure what merge is off, but it looks like the `timeout` stuff is lost here...